README.md

```diff
@@ -87,14 +87,20 @@ Selecting the format you'd like your project to be exported as while choosing th
 
 ![Alt Text](https://media.giphy.com/media/I5g06mUnVzdX7iT2Gf/giphy.gif)
 
-## Deving locally
-1. clone repo
-2. create a virtual env: `virtualenv local_dev`
-3. activate virtual env: `source local_dev/bin/activate`
-4. install packages from repo: `pip3 install -e ".[dev]"`
-5. create script on top level of `ROBOFLOW-PYTHON` and reference local `roboflow` directory: `from roboflow import Roboflow`
-6. when done, uptick the pip package minor version number in `setup.py`
-7. manually add any new dependencies to the `requirements.txt` and list of dependencies in `setup.py` (careful not to overwrite any packages that might screw up backwards dependencies for object detection, etc.) 
+## Developing locally
+
+1. clone this repo
+1. with docker
+    1. `docker build -t roboflow`
+    1. copy `.env-example` to `.env` and update environment variables for testing
+    1. shell into container: `docker run -it --rm --env-file .env -v /your/path/to/roboflow-python:/roboflow roboflow`
```

Comment:
I think you need to map the current user with the container

Reply:
what purpose does this serve?

Reply:
to ensure everything written inside the container can be accessed by the current host user. E.g. maybe I am logging something to a file, once I close the container I won't be able to edit it, since it was created by a different user